server/rooms.js

```diff
@@ -23,7 +23,7 @@ router.get('/:roomId', async (req, res, next) => {
   try {
     let { roomId } = req.params;
     roomId = parseInt(roomId, 10) + roomIdAdjustment;
-    if (!(tempStorage.roomInfo.id && tempStorage.roomInfo.id === roomId)) {
+    if (!(tempStorage.roomInfo.id || tempStorage.roomInfo.id === roomId)) {
```

Comment:
Just out of curiosity, how did this fix the bug? I haven't worked with temporary storage on my server yet. Is roomInfo.id referencing/storing the last query to your database? Would it be undefined at times (e.g. on the first request?)

Reply:
oh, my intension was to retrieve just from memory when (tempStorage.roomInfo.id matches roomId), so if there's no tempStorage.roomInfo.id, I need to perform this step, which is to query database. If there's tempStorage.roomInfo.id, but tempStorage.roomInfo.id is not the same as roomId, I still need to perform this step.